lethal-squirrels/Bangazon-Site
# Matt cancel order

## REFERENCE
*** Merge after Shopping Cart View is Merged into master
This merge utilizes the Shopping Cart View
Ticket: (#9)

## DESCRIPTION
What did you do? Why did you do it?
I added functionality to cancel an order which removes the order as well as all associated ProductOrder rows.

## STEPS TO TEST
1. Clone this branch to a local repo
2. Run the program
3. Test functionality
	* View your shopping cart
	* Cancel the order
	* Ensure all Relevant rows are removed
